app/js/target-disease-controller.js

```diff
@@ -107,9 +107,19 @@
                     is_open : false,
                     is_loading: false,
                     heading : cttvDictionary.LITERATURE,
-                    source : cttvConfig.evidence_sources.literature,
-                    source_label : cttvConfig.evidence_sources.literature.map(function(s){return {label:cttvDictionary[ cttvConsts.invert(s) ], url:cttvConsts.dbs_info_url[cttvConsts.invert(s)]}; }),
+                    source : cttvConfig.evidence_sources.literature[0],
+                    source_label : cttvConfig.evidence_sources.literature.map(function(s){ if (s !== 'NFERX') {
+                        return {label:cttvDictionary[ cttvConsts.invert(s) ], url:cttvConsts.dbs_info_url[cttvConsts.invert(s)]}; }} ),
                     has_errors: false,
+                    nferx : {
```

Comment:
I don't feel like having all this (biogen) specific datasources in the public code looks right. We should move to implement plugins for the evidence page.

Reply:
👍 i agree. the hope is that the datasources will be of common interest to drug partner. but the overlap will never be perfect. 

do you want to wait until we have plugins or include for now anyways?

Reply:
I suppose we will need to live with this for now, but try to move to evidence plugins asap